Assets/Code/Scripts/PlayerManagement/CameraManager.cs

```diff
@@ -11,16 +11,23 @@ public class CameraManager : MonoBehaviour
     private RealityMovement _realPlayerMovement;
     private MouseLook _realMouseLook;
     
-    [SerializeField] private GameObject _noClipPlayerCamera;
-    private MouseLook _noClipMouseLook;
+    [SerializeField] private GameObject _noclipPlayer;
+    [SerializeField] private GameObject _noclipCamera;
+    private NoclipMovement _noclipMovement;
+    private MouseLook _noclipMouseLook;
 
+
+    [SerializeField] private KeyCode _activation = KeyCode.P;
```

Comment:
This is not used anymore, I think it's clearer if we remove it

Reply:
yeah I agree, I haven't see this thing